### ISSUE TITLE
Test napari org plugins (e.g. napari-animation)

### ISSUE THREAD

**Issue by psobolewskiPhD** [napari/napari#6309]
## 🧰 Task

Test napari org plugins (e.g. napari-animation, napari-svg) either on PR or as part of comprehensive or as a cron.
The idea is to improve visibility if a change to napari main breaks napari plugins. As a side effect, this could surface issues that could be affecting plugin developers.

**Comment by Czaki** [comment]
hmm... 

We currently totally ignore failing comprehensive tests. adding next failing test will not solve problem without person who will trace such problems.

**Comment by dalthviz** [comment]
Maybe creating/defining a workflow over the plugins repos (could be set as a cron that runs at the end of the day or week installing napari from the latest main) could help detect incompatibilities while preventing adding more complexity to the workflows definitions in this repo? In that way I think it could be possible to detect that a plugin has compatibility issues early (which makes a lot of sense) without changing things here. Also, I think people that could be actually interested in a particular plugin could jump in an easier way to fix the detected errors (since the failed check will be shown in the actual plugin repo). Basically following the initial idea here but making the compatibility errors detection something to be managed by each plugin instead of centralizing it over the main napari repo

**Comment by GenevieveBuckley** [comment]
> making the compatibility errors detection something to be managed by each plugin instead of centralizing it over the main napari repo

I agree with dalthviz, this is better achieved by adding an "upstream" CI job to the repositories that depend on napari. A cron job could run these at semi-regular intervals, and install napari from the current main branch here.

If you suggested this in part because you wanted failures to be linked back to the specific PR in napari, git-bisect is a better fit for that.
